File: Exercicios_guanaraba/Com_Ruby/exercicios/ex017_Catetos_e_Hipotenusa.rb
=begin

Faça um programa que leia o comprimento do cateto oposto e do cateto adjacente
de um triângulo retângulo, calcule e mostre o comprimento da hipotenusa.

=end
puts 'Digite o cateto oposto: '
co = gets.chomp.to_f
puts 'Digite o cateto adjacente: '
ca = gets.chomp.to_f

hipo = (co**2 + ca**2) ** (0.5)

puts "A hipotenusa é: #{hipo.truncate(12)}"

File: Exercicios_guanaraba/Com_Dart/bin/ex017_Catetos_e_Hipotenusa.dart
/*

Faça um programa que leia o comprimento do cateto oposto e do cateto adjacente de um 
triângulo retangulo, calcule e mostre o comprimento da hipotenusa.

*/
import 'dart:io';
import 'dart:math';

void main(){

  print ('Digite o cateto oposto: ');
  double co = double.parse(stdin.readLineSync());

  print ('Digite o cateto adjacente: ');
  double ca = double.parse(stdin.readLineSync());

  double hipo = pow((pow(co, 2) + pow(ca, 2)), 0.5);

  print ('A hipotenusa é: $hipo');


}